### PR TITLE
Added new optiSLang integration test scenario

### PR DESCRIPTION
**Pull request by egravenh**

### CODE REVIEW

**Comment by seanpearsonuk** [comment]
Hi @egravenh, would it be OK for me to add someone from optiSLang at this stage?

File: tests/test_optislang/test_optislang_integration.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - Input parameters
+    - Output parameters
+    """
+
+    # Step 1: Setup logging 
+    pyfluent.set_log_level('ERROR')
+
+    # Step 2: Launch fluent session and read case file with and without data file
+    session = load_mixing_elbow_param_case_dat
+    session.check_health() == 'SERVING'
```
Comment (seanpearsonuk) [inline tests/test_optislang/test_optislang_integration.py:32]: This line will actually have no effect as it stands.
```suggestion
    assert session.check_health() == 'SERVING' 
```

**Comment by egravenh** [comment]
Hi @seanpearsonuk, please do.

File: tests/test_optislang/test_optislang_integration.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    else: 
+        session.solver.tui.solve.iterate()
+
+    convergence = session.scheme_eval.string_eval("(rpgetvar 'solution/converged?)")
+
+    # solution output (test conditional statement)
+    if convergence == "#f": # -> let user know 
+        print('Failed to converge')
+    else:
+        print('Solution is converged')
```
Comment (seanpearsonuk) [inline tests/test_optislang/test_optislang_integration.py:75]: We don't really require this kind of output in the PyFluent tests, and it's taking up four lines.

Let's be succinct - see my next comment.

File: tests/test_optislang/test_optislang_integration.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    assert convergence == "#t"
```
Comment (seanpearsonuk) [inline tests/test_optislang/test_optislang_integration.py:77]:
```suggestion
    assert convergence == "#t", "Solution failed to converge"
```

**Review by tejalprabhu01 — APPROVED** [review]

File: tests/test_optislang/test_optislang_integration.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    # Step 2: Launch fluent session and read case file with and without data file
+    session = load_mixing_elbow_param_case_dat
+    assert session.check_health() == 'SERVING' 
+    case_path = str(Path(pyfluent.EXAMPLES_PATH) / "elbow_param.cas.h5")
+    session.solver.tui.file.read_case_data(case_path)
+
+    #################################################################
+    # Step 3: Get input and output parameters and create a dictionary 
+    from ansys.fluent.core.services.scheme_eval import Symbol
```
[the remark this answers is not in the document]
Comment (egravenh) [reply tests/test_optislang/test_optislang_integration.py]: Hi @tejalprabhu01, as discussed in our call, we will add 23.1 updates to the test plan and implement once automation is switched over. Thanks!

File: tests/test_optislang/test_optislang_integration.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    output_parameters = session.scheme_eval.eval((Symbol("list-output-parameters"),)) 
+    input_parameters = input_parameters['inlet2_temp']
+    output_parameters = output_parameters['outlet_temp-op']
+    session.solver.tui.file.read_case(case_path)
+
+    #################################################################
+    # Step 4: Set a variation on these input parameters 
+    # variations/designs are generated by optiSLang based on 
+    # algorithm selected
+    session.solver.tui.define.parameters.input_parameters.edit(
```
[the remark this answers is not in the document]
Comment (egravenh) [reply tests/test_optislang/test_optislang_integration.py:47]: Same as above.

**Review by ypatel-qa — APPROVED** [review]

**Pull request merged**